lib/Dialect/Handshake/HandshakeOps.cpp

```diff
@@ -1729,6 +1742,9 @@ void BundleOp::build(OpBuilder &odsBuilder, OperationState &odsState,
                      ChannelType channelType) {
   assert(isa<handshake::ControlType>(ctrl.getType()) &&
          "expected !handshake.control");
+  assert(cast<handshake::ControlType>(ctrl.getType()).getNumExtraSignals() ==
```

Comment:
`assert(!cast<handshake::ControlType>(ctrl.getType()).getNumExtraSignals(), "expected ctrl to have no extra signals")`

Reply:
I think `== 0` is clearer Why did you change it?

Reply:
It's just because LLVM code generally relies on the autoconversion from number to bool (where 0 is false and everything else is true) in those kind of cases. It helps with readability when the code "looks the same" everywhere. But not a big deal really, just for future reference.